app/views/users/edit.html.erb

```diff
@@ -28,7 +28,24 @@
   <% @schedules.each do |schedule| %>
     <tr>
       <td><span class="glyphicon glyphicon-calendar" aria-hidden="true"></span> <%= "#{schedule.title}" %></td>
-      <td><%= link_to edit_user_schedule_path(@guider, schedule), title: "Edit Schedule #{schedule.title}", class: 'btn btn-info' do %><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span><span class="sr-only">Edit Schedule</span><% end %></td>
+      <td nowrap="true">
+        <%= link_to edit_user_schedule_path(@guider, schedule), title: "Edit Schedule #{schedule.title}", class: 'btn btn-info' do %><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span><span class="sr-only">Edit Schedule</span><% end %>
+        <% if schedule.destroyable? %>
+          <%= link_to(
+              user_schedule_path(@guider, schedule),
+              method: :delete,
+              title: 'Delete Schedule',
+              class: 'btn btn-danger t-delete',
+              data: {confirm: 'Are you sure you want to delete this schedule?'}
```

Comment:
The house-style is to include spaces in the hash braces. Also, we should confirm with the title of the schedule so they know if they've clicked the wrong one.